pkg/monitoring/metrics/forklift-controller/metrics.go

```diff
@@ -0,0 +1,60 @@
+package forklift_controller
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+const (
+	Succeeded = "Succeeded"
+	Failed    = "Failed"
+	Warm      = "Warm"
+	Cold      = "Cold"
+	Local     = "Local"
+	Remote    = "Remote"
+)
+
+var (
+	// 'status' - [ succeeded, failed ]
+	// 'provider' - [oVirt, vSphere, Openstack, OVA, openshift]
+	migrationGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "mtv_workload_migrations",
+		Help: "VM Migrations sorted by status and provider type",
+	},
+		[]string{"status", "provider"},
+	)
+)
+
+var (
+	// 'status' - [ succeeded, failed ]
+	// 'provider' - [oVirt, vSphere, Openstack, OVA, openshift]
+	planStatusGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "mtv_workload_plans",
+		Help: "VM migration Plans sorted by status and provider type",
+	},
+		[]string{
+			"status",
+			"provider",
+		},
+	)
+
+	// 'type' - [ cold, warm ]
+	planTypeGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "mtv_workload_plans_type",
+		Help: "VM migration Plans type",
+	},
+		[]string{
+			"type",
```

Comment:
maybe there is a better name for it? `type` doesn't really tell me it's gonna be cold/warm

Reply:
maybe switch it to `warm` and then have like a boolean `true/false` to determine?

Reply:
I'd use `mode`